langchain4j-open-ai/src/test/java/dev/langchain4j/model/openai/OpenAiTokenizerIT.java

```diff
@@ -34,6 +48,7 @@
 
 // TODO use exact model for Tokenizer (the one returned by LLM)
 @Disabled("this test is very long and expensive, we will need to set a schedule for it to run maybe 1 time per month")
+@EnabledIfEnvironmentVariable(named = "OPENAI_API_KEY", matches = ".+")
```

Comment:
it is already disabled

Reply:
@langchain4j  Do we want to enable it conditionally (remove `@Disabled`)?

Reply:
let's keep it disabled for now please